app/models/katello/glue/provider.rb

```diff
@@ -95,11 +95,13 @@ def owner_upstream_update(upstream, _options)
         #ca_file = '/etc/candlepin/certs/upstream/subscription.rhn.stage.redhat.com.crt'
         ca_file = nil
 
-        capabilities = Resources::Candlepin::CandlepinPing.ping['managerCapabilities'].inject([]) do |result, element|
+        params = {}
+        params[:capabilities] = Resources::Candlepin::CandlepinPing.ping['managerCapabilities'].inject([]) do |result, element|
           result << {'name' => element}
         end
+        params[:facts] = {:distributor_version => 'sat-6.3'}
```

Comment:
I'm fine with this change for now, but looking ahead should this be a configuration value that lives in either application `Settings` or system configuration `SETTINGS` to make it easier to swap it on a given instance?